dev/com.ibm.ws.security.acme/resources/OSGI-INF/l10n/metatype.properties

```diff
@@ -28,17 +28,11 @@ validFor.desc=The duration of time that the certificate signing request specifie
 subjectDN=Subject distinguished name
 subjectDN.desc=Subject distinguished name (DN) to use for the certificate. The DN can include the following relative distinguished name (RDN) types: cn, c, st, l, o and ou. If the cn RDN type is defined, it must be one of the domains defined by the domain configuration element and it must be the first RDN in the DN. If the cn RDN type is not defined, the first domain defined by the domain configuration element is used as the cn RDN value.
 
-challengeRetries=Challenge retries
-challengeRetries.desc=The number of times to retry updating the challenge status before aborting the challenge.
+challengePollTimeout=Challenge poll timeout
+challengePollTimeout.desc=The amount of time to spend polling the status of an ACME challenge before aborting.
```

Comment:
Suggested update:
"The amount of time to spend polling the status of an ACME challenge before the challenge cancels."

Reply:
Talked to Charlotte on slack, we decided on the following:

`The amount of time to spend polling the status of an ACME challenge before polling discontinues and treats the challenge as failed.`